Source/Library/GameBoardWindow/GameBoardController.cpp

```diff
@@ -0,0 +1,33 @@
+#include "TicTacToe/Library/GameBoardWindow/GameBoardController.hpp"
+
+using namespace boost::signals2;
+using namespace TicTacToe;
+
+GameBoardController::GameBoardController(int x_badge_count, int o_badge_count)
+    : m_game_board_window(m_game_model.get_board(), Board::Token::X),
+      m_game_state(GameModel::State::CONTINUE) {
+  m_game_board_window.get_x_badge_panel().set_score(x_badge_count);
+  m_game_board_window.get_o_badge_panel().set_score(o_badge_count);
+  m_game_board_window.get_panel().connect_clicked_signal([this] (int x, int y) {
```

Comment:
Prefer using a method instead of inlining. This allows different logical parts of the codebase to be separated instead of having massive constructors all containing separate inline game logic.

Reply:
I am a little confused. Do you mean that I should add the set_score member function or other member functions into the GameBoardController class?

Reply:
Add another member function to implement this logic. Keep the constructor minimal so that all it does is initialization.